src/main/resources/META-INF/plugin.xml

```diff
@@ -4,10 +4,22 @@
     <vendor email="support@jfrog.com" url="https://www.jfrog.com">JFrog</vendor>
 
     <description><![CDATA[
-      The JFrog plugin adds JFrog Xray scanning of Maven, Gradle, Go, npm, Yarn v1, and PyPI project dependencies to your IDE.
-      The plugin allows developers to see security vulnerabilities and license violations within their code by continuously scanning it locally with JFrog Xray.
-      It also allows developers to track the status of the code while it is being built, tested and scanned on the CI server.
-      The plugin is compatible with IntelliJ IDEA, WebStorm, GoLand, PyCharm, and Android Studio.
+        The plugin allows developers to find and fix security vulnerabilities in their projects and to see valuable information about the status of their code by continuously scanning it locally with JFrog Xray.
+
+        Software Composition Analysis (SCA)
+        Scan your project dependencies for security issues.
```

Comment:
I suggest mentioning that the plugin also lets the user update vulnerable dependencies.